### DISCUSSION TITLE
How to install and use

### DISCUSSION THREAD

**Discussion by Kaffeedor** [ccostes/rtl-sdr-rs#13] [Q&A] [answered]
Hello there,
I am new to Rust (and generally to low level languages) and I would like to know how do I actually install and use this libary in my programm?
Also, does it work on Windows (if not I'm just going to use my RPi or a VM)
Thanks for an answer in advance and Greetings,
Murphy

**Comment by ghost** [comment]
Hi, 

it always depends what you want to achieve. There is the 'official'  osmocom lib. See https://github.com/osmocom/rtl-sdr. This version is a Rust port and therefore interesting, since no c-bindings is required.

Regards, 
Jiri

**Comment by Kaffeedor** [comment, marked as answer]
I figured it out, I just downloaded the whole rep., It seems to be working on windows and detecing my SDR (its a nooelec nesdr smart gen4 i think).
Now I wanted to try out that FM example, the Commands to run it that are provided are only for Mac/Linux tho. So I tried searching for something similar on Windows and I am using the following now:
`cargo run --example simple_fm | vlc --rawaud-samplerate=32000 --audiofile-format=S16_LE`, which should be the same as the Linux one provided just with "vlc" (you need to download VLC and then just add the folder of it to path).
It seems to be working and I get no errors BUT: It doesnt play any audio, I do not know enough about this to find out where the error might be, if anyone could help with that would be amazing. I am gonna play around with it for a while now, maybe I will find something...
